hack/lib.sh

```diff
@@ -76,7 +79,6 @@ function hack::ensure_kubectl() {
     if hack::verify_kubectl; then
         return 0
     fi
-    echo "Installing kubectl v$KUBECTL_VERSION..."
     tmpfile=$(mktemp)
```

Comment:
If I already installed kubectl 1.17.0 in my env, but the requested kubectl version is 1.12.0, will it replace the kubectl in my env

Reply:
it won't. The ensuring only install the bin in the $project/output/bin.

Reply:
Make sense. Thanks for the clarification.